File: trail1/물의 상태/state-of-water.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        if (N < 0) {Console.WriteLine("ice");}
        else if (N >= 100) {Console.WriteLine("vapor");}
        else {Console.WriteLine("water");}
    }
}

File: trail1/물의 상태/state-of-water.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    if (n < 0) {cout << "ice" << "\n";}
    else if (n >= 100) {cout << "vapor" << "\n";}
    else {cout << "water" << "\n";}    
    return 0;
}